pymc3/tests/test_distributions.py

```diff
@@ -2944,3 +2967,25 @@ def func(x):
     import pickle
 
     pickle.loads(pickle.dumps(y))
+
+
+def test_distinct_rvs():
```

Comment:
Does this make more sense in test model?

Reply:
Not so much; the relevant logic is actually implemented in `Distribution.dist`.